ddtrace/contrib/django/patch.py

```diff
@@ -77,20 +77,20 @@
 
 
 _NotSet = object()
-psycopg_cursor_cls = Psycopg2TracedCursor = _NotSet
+psycopg_cursor_cls = PsycopgTracedCursor = _NotSet
 
 
 def patch_conn(django, conn):
```

Comment:
should we add additional testing for this, to make sure that we pick out we instrument correctly whether its psycopg or pscyopg2?